src/plugins/recordTypes/hit/fields.js

```diff
@@ -0,0 +1,855 @@
+            },
+          },
+          handlingLimitationsGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.hits_common.handlingLimitationsGroup.name',
+                  defaultMessage: 'Handling limitations',
+                },
+              }),
+              repeating: true,
+              view: {
+                type: CompoundInput,
+              },
+            },
+            handlingLimitationsType: {
+              [config]: {
+                messages: defineMessages({
+                  fullName: {
+                    id: 'field.hits_common.handlingLimitationsType.fullName',
+                    defaultMessage: 'Handling limitations type',
+                  },
+                  name: {
+                    id: 'field.hits_common.handlingLimitationsType.name',
+                    defaultMessage: 'Type',
+                  },
+                }),
+                view: {
+                  type: TermPickerInput,
+                  props: {
+                    source: 'handlinglimitationstypes',
+                  },
+                },
+              },
+            },
+            handlingLimitationsRequestor: {
+              [config]: {
+                messages: defineMessages({
+                  fullName: {
+                    id: 'field.hits_common.handlingLimitationsRequestor.fullName',
+                    defaultMessage: 'Handling limitations requestor',
+                  },
+                  name: {
+                    id: 'field.hits_common.handlingLimitationsRequestor.name',
+                    defaultMessage: 'Requestor',
+                  },
+                }),
+                view: {
+                  type: AutocompleteInput,
+                  props: {
+                    source: 'person/local,organization/local',
+                  },
+                },
+              },
+            },
+            handlingLimitationsLevel: {
+              [config]: {
+                messages: defineMessages({
+                  fullName: {
+                    id: 'field.hits_common.handlingLimitationsLevel.fullName',
+                    defaultMessage: 'Handling limitations level',
+                  },
+                  name: {
+                    id: 'field.hits_common.handlingLimitationsLevel.name',
+                    defaultMessage: 'Level',
+                  },
+                }),
+                view: {
+                  type: TermPickerInput,
+                  props: {
+                    source: 'handlinglimitationslevels',
+                  },
+                },
+              },
+            },
+            handlingLimitationsOnBehalfOf: {
+              [config]: {
+                messages: defineMessages({
+                  fullName: {
+                    id: 'field.hits_common.handlingLimitationsOnBehalfOf.fullName',
+                    defaultMessage: 'On behalf of',
```

Comment:
```suggestion
                    defaultMessage: 'Handling limitations on behalf of',
```

I _think_ this is in the handling limitations group, so just clean up the full name message.